### ISSUE TITLE
Support Vue 2.0

### ISSUE THREAD

**Issue by simplesmiler** [simplesmiler/vue-focus#7]
The 2.0 release of Vue draws near, and `vue-focus` should follow.

Besides internal reworkings, 2.0 will require some changes to `vue-focus` API. Namely, with two-way directives now gone, `v-focus-model` becomes obsolete, and should be replaced with something like:

``` html
<input v-focus="expr" @focus="expr = true" @blur="expr = false">
```

**Comment by simplesmiler** [comment]
The `v-focus-auto` will probably be gone too, because `v-focus="true"` does the same while being more flexible.

**Comment by reinerBa** [comment]
Hi there, is the vue-focus v2 still compatible with vue 1 or may i only use it with vue 2?

**Comment by simplesmiler** [comment]
No. With directives API changing in Vue 2.0 there is no good way to support 1.x and 2.x with a single codebase.

Although you will still be able to `npm install vue-focus@^1.0` to get Vue 1 compatible version.

**Comment by PrimozRome** [comment]
@simplesmiler if I want to use this with Vue 2.0 do I need to install the 2.0 brach? Is it available on npm already?

**Comment by simplesmiler** [comment]
@PrimozRome it is on npm as `npm install vue-focus@next`. It will complain about peer dependency because Vue 2.0 is not released yet, but should work just fine. Once Vue 2.0 goes live, `npm install vue-focus` will install 2.0-compatible version.

**Comment by simplesmiler** [comment]
Published 2.0, see [release notes](https://github.com/simplesmiler/vue-focus/releases/tag/2.0.0).
